client/asset/dcr/wallet.go

```diff
@@ -143,7 +145,19 @@ type Wallet interface {
 	PeerCount(ctx context.Context) (uint32, error)
 	// AddressPrivKey fetches the privkey for the specified address.
 	AddressPrivKey(ctx context.Context, address stdaddr.Address) (*secp256k1.PrivateKey, error)
-
+	// StakeDiff gets the stake difficulty.
+	StakeDiff(ctx context.Context) (dcrutil.Amount, error)
+	// PurchaseTickets purchases n tickets. vsp arguments only needed for
+	// internal wallets.
+	PurchaseTickets(ctx context.Context, n int, vspHost, vspPubKey string) ([]string, error)
```

Comment:
Please can we specify the exact arguments that are not needed for `RPC` wallets? I see only `vspHost` and `vspPubKey` are ignored and all other arguments are provided. `vsp arguments` might not ring a bell the first time.

Reply:
`vspHost and vspPubKey only needed for internal wallets.` okay?